Code/Framework/AzToolsFramework/AzToolsFramework/Asset/AssetSeedManager.cpp

```diff
@@ -645,6 +650,10 @@ namespace AzToolsFramework
         if (useDebugInfoList)
         {
             debugInfo.BuildHumanReadableString();
+            if (debugInfo.m_invalidFileDebugInfoList.size() > 0)
```

Comment:
For code intent purposes. a check of !.empty() can be read a bit clearer as , "buld the Invalid Asset Human Readable string, if the invalid file debug list is not empty.
```suggestion
            if (!debugInfo.m_invalidFileDebugInfoList.empty())
```